README.rst

```diff
@@ -16,6 +18,9 @@ We benchmark the following methods:
 * FastSRM
 * Optimal Transport
 * FUGW
+* Hyperalignment : 
+  - pcha : Parcellation Hyperalignment
```

Comment:
There is no benefit in using `pcha` and `slha`, please rename them to `parcellation_hyper_alignment` etc